From: Alexey Dobriyan <adobriyan@gmail.com>
Subject: proc: mark /proc/cmdline as permanent
Date: Fri, 30 Dec 2022 12:36:02 +0300

/proc/cmdline is never removed, mark is as permanent for slightly faster
open and close.

Link: https://lkml.kernel.org/r/Y66xAveh2yUsP7m9@p183
Signed-off-by: Alexey Dobriyan <adobriyan@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 fs/proc/cmdline.c |    1 +
 1 file changed, 1 insertion(+)

--- a/fs/proc/cmdline.c~proc-mark-proc-cmdline-as-permanent
+++ a/fs/proc/cmdline.c
@@ -17,6 +17,7 @@ static int __init proc_cmdline_init(void
 	struct proc_dir_entry *pde;
 
 	pde = proc_create_single("cmdline", 0, NULL, cmdline_proc_show);
+	pde_make_permanent(pde);
 	pde->size = saved_command_line_len + 1;
 	return 0;
 }
_
